plugin/pkg/admission/resourcequota/config.go

```diff
@@ -37,18 +37,39 @@ func init() {
 	install.Install(scheme)
 }
 
+func defaultAdmissionConfig() *resourcequotaapi.Configuration {
+	return &resourcequotaapi.Configuration{
+		LimitedResources: []resourcequotaapi.LimitedResource{
+			{
+				Resource: "pods",
+				MatchScopes: []corev1.ScopedResourceSelectorRequirement{
+					{
+						ScopeName: corev1.ResourceQuotaScopePriorityClass,
+						Operator:  corev1.ScopeSelectorOpIn,
+						Values:    []string{"system-cluster-critical", "system-node-critical"},
+					},
+				},
+			},
+		},
+	}
+}
+
 // LoadConfiguration loads the provided configuration.
 func LoadConfiguration(config io.Reader) (*resourcequotaapi.Configuration, error) {
 	// if no config is provided, return a default configuration
-	if config == nil {
+	/*if config == nil {
```

Comment:
The problem with this approach is if `cluster-admin` gives an admission config, we cannot have this default admission. config being used. Alternate approaches that I can think of:

- Read the config and merge it with default config, we have to be careful while merging because there is a chance that `cluster-admin` purposefully sets up a config which is contradicting the default admission config. The downside of this approach is it's not transparent to the user.